src/Scanner.js

```diff
@@ -231,6 +231,9 @@ class DefaultScanner {
             return
         }
 
+        // Do not add explicitly search forms
+        if (parentForm.role === 'search') return
```

Comment:
Just occurred to me we shouldn't even add these obvious `search` forms, whenever web authors are kind enough to mark them as such. Now there's a new `search` html element, but not worth bothering about it now.